src/core/Entry.cpp

```diff
@@ -508,7 +508,8 @@ Entry* Entry::clone(CloneFlags flags) const
         entry->m_data.timeInfo.setLocationChanged(now);
     }
 
-
+    if (flags & CloneRenameTitle)
+        entry->setTitle(entry->title() + tr(" - Clone"));
```

Comment:
Good idea.  I often wonder after cloning an entry, that which of two I'm seeing is the original which keeps the history.